deployment/docker/provisioning/datasources/victorialogs-datasource/victorialogs-datasource.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: 1
+
+datasources:
+  - name: VictoriaLogs
+    type: victorialogs-datasource
+    access: proxy
+    url:  http://victorialogs:9428
+
+  - name: VictoriaMetrics
+    type: prometheus
+    access: proxy
+    url: http://victoriametrics:8428
```

Comment:
In the Grafana dashboards, I saw an error like the one in the image below. Also, I can't choose the data source from the dropdown. 

<img width="1680" alt="Screenshot 2024-02-25 at 15 39 26" src="https://github.com/VictoriaMetrics/VictoriaMetrics/assets/12189570/f3692380-81da-45d0-903f-86724335fd9b">

Reply:
Thanks! This should be addressed in https://github.com/VictoriaMetrics/VictoriaMetrics/pull/5857/commits/97f31d89c96314aaa925c8c128fcf8e19891a47a